scripts/relaunch_kmonad_after_keycode_error.ps1

```diff
@@ -0,0 +1,115 @@
+# IMPORTANT: 
+#
+# 1. Set $kmonadPath and $kbdConfigPath below
+#
+# 2. BE SURE kbdConfigPath has '" and "' on either end and NOT JUST normal single quotes or normal double quotes.
+#
+# 3. IF your kmonad executable filename doesn't match *kmonad* then go to the finally block at the end of this file
+#         and update the pattern so that it will match and kill your kmonad process if the user hits Ctrl + C on this script.
+
+
+# -------------========================= SCRIPT GENERAL INFORMATION =========================-------------
+#
+# This powershell script launches kmonad, then watches for the error message 
+#       "Encountered error in KeySource: Cannot translate from windows keycode" which means the kmonad process has crashed.
+#
+# If it sees this, it immediately kills the existing kmonad process and then launches a new one, again watching for a crash.
+#
+# It also handles Ctrl + C just fine and will kill any existing kmonad processes before exiting as long as the matching pattern is 
+#       correctly in the "finally" block at the end of this file.
+
+
+try
+{
+    while($true)
+    {
+        Write-Host "Starting new kmonad process"
+    
+        $blockToRun = 
+        {        
+            $kmonadPath = "D:\Kmonad-Windows\kmonad-key-repeat-fixed.exe"
+            $kbdConfigPath = '"D:\source\repos\linux-files\configuration\kmonad\david_default.kbd"'
```

Comment:
Perhaps we should set these to definitely non-existent files, so this just results in an error if the user forgot to set these. Also, wouldn't it make more sense to make these CLI arguments?